pkgs/top-level/all-packages.nix

```diff
@@ -7779,6 +7779,8 @@ with pkgs;
 
   yodl = callPackage ../development/tools/misc/yodl { };
 
+  yq = callPackage ../development/tools/yq { };
```

Comment:
Please use `python3Packages.callPackage` and take `buildPythonApplication` and `pyyaml` as arguments to the function instead.